### DISCUSSION TITLE
如何加载sdcard的js或者kbc1文件？

### DISCUSSION THREAD

**Discussion by snice** [openkraken/kraken#1118] [Q&A]
如何加载sdcard的js或者kbc1文件？

**Comment by cnryb** [comment]
看这里  
https://github.com/openkraken/kraken/blob/0f1701d43f2f0999cbd941deca7911ec76682d45/kraken/lib/src/launcher/bundle.dart#L149-L158

**Comment by wssgcg1213** [comment]
通过路径加载即可, 本地使用 file 协议
